src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/ModuleForceUnicodeChat.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2016 - 2021 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.ccbluex.liquidbounce.features.module.modules.exploit
+
+import net.ccbluex.liquidbounce.event.PacketEvent
+import net.ccbluex.liquidbounce.event.handler
+import net.ccbluex.liquidbounce.features.module.Category
+import net.ccbluex.liquidbounce.features.module.Module
+import net.minecraft.network.packet.c2s.play.ChatMessageC2SPacket
+
+object ModuleForceUnicodeChat : Module("ForceUnicodeChat", Category.EXPLOIT) {
+
+    val packetHandler = handler<PacketEvent> { event ->
+        if (event.packet is ChatMessageC2SPacket) {
+            val chatMessage = event.packet
```

Comment:
This might be working, but is not really well written code. Might try to make it better instead of pasting it from the old liquidbounce.